src/fights.ts

```diff
@@ -379,6 +385,16 @@ function embezzlerSetup() {
   if (!get("_cameraUsed") && !have($item`shaking 4-d camera`)) {
     retrieveItem($item`4-d camera`);
   }
+
+  // Fix invalid spooky putty monster (caused by ascending)
+  if (have($item`Spooky Putty monster`) && get("spookyPuttyMonster") === $monster`none`) {
```

Comment:
A similar thing can happen to rain-doh if you use the copier while wearing papier-mache or other text-obfuscating conditions--is it worth adding a refresher there?